### ISSUE TITLE
Функция оценки обученности модели evaluate_model

### ISSUE THREAD

**Issue by GeekNekoS** [GeekNekoS/Chara#12]
Необходимо написать определение функции def evaluate_model...
Аргументы:
model: keras.Model - обученная модель
test: tf.Dataset - датасет для тестирования
Вывод:
None
1. Функция выводит confusion_matrix
2. Accuracy
3. Precision
4. Recall
5. F1
6. графики loss и accuracy обучения по эпохам

Если будет не хватать входных данных обсудить здесь
